test/blackbox-tests/test-cases/dune-init.t/run.t

```diff
@@ -369,7 +369,7 @@ We can build the project:
 
 And the opam file will be generated as expected
 
-  $ cat test_exec_proj/test_exec_proj.opam
+  $ cat test_exec_proj/test_exec_proj.opam | sed 's/"dune"/$dune/'
```

Comment:
Why do you change to dollar here?

Reply:
Just to demonstrate that it's a variable in the output